### PR TITLE
#7793: Created migrate module from custom csv.

### PR DESCRIPTION
**Pull request by aster06**

### CODE REVIEW

File: web/modules/custom/migration_csv/assets/users_registration.csv [hunk 1]
```diff
@@ -0,0 +1,4 @@
+uid,country,user_city
```
Comment (faust174) [inline web/modules/custom/migration_csv/assets/users_registration.csv:1]: Probably custom user data should be in the same CSV file as default registration data or at least it should have the same uid's.

**Review by faust174 — APPROVED** [review]

File: web/modules/custom/migration_csv/src/Plugin/migrate/destination/CustomDestination.php [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(
+    ContainerInterface $container,
+    array $configuration,
+    $plugin_id,
+    $plugin_definition,
+    MigrationInterface $migration = NULL): CustomDestination|ContainerFactoryPluginInterface|static {
```
Comment (lavrenchuk-dev) [inline web/modules/custom/migration_csv/src/Plugin/migrate/destination/CustomDestination.php]: ?

File: web/modules/custom/migration_csv/migrations/migration_csv_users.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  ids:
+    - uid
+process:
+  name: username
+  mail: email
+  status: status
+  roles: roles
+  type:
+    plugin: default_value
+    default_value: article
```
Comment (lavrenchuk-dev) [inline web/modules/custom/migration_csv/migrations/migration_csv_users.yml]: article?

File: config/default/core.extension.yml [hunk 4]
```diff
@@ -50,12 +50,18 @@ module:
...
+  migrate_drupal_ui: 0
+  migrate_source_csv: 0
+  migration_csv: 0
   mysql: 0
   node: 0
   options: 0
   page_cache: 0
   path: 0
   path_alias: 0
+  phpass: 0
```
Comment (lavrenchuk-dev) [inline config/default/core.extension.yml:64]: why do you need this?

**Review by lavrenchuk-dev — CHANGES_REQUESTED** [review]

File: config/default/core.extension.yml (hunk 4 above)
Comment (aster06) [reply config/default/core.extension.yml:64]: Module Migrate Drupal requires this

**Review by lavrenchuk-dev — APPROVED** [review]

**Pull request merged**